### ISSUE TITLE
Unnecessary error displayed in homepage when there are no episodes on the website to display

### ISSUE THREAD

**Issue by ademola-raimi** [andela/suyabay#193]
It was observed that Unnecessary error messages do pop up if the home page is visited when there are no episodes on the website. The error messages are shown below:

![screen shot 2016-03-06 at 4 56 27 pm](https://cloud.githubusercontent.com/assets/14159420/13555392/bca08650-e3bf-11e5-9cdb-05d92d498fc2.png)

**Comment by unicodeveloper** [comment]
This should be a bug not a feature

**Comment by ademola-raimi** [comment]
You are right, thanks

**Comment by unicodeveloper** [comment]
I am still waiting on the PR for this, how has this issue not been fixed yet? @andela-araimi

**Comment by ademola-raimi** [comment]
I have fixed it, I thought you have merged it already. I will send another pull request with that branch

**Comment by unicodeveloper** [comment]
Then check the staging branch to be sure it's been merged before

**Comment by ademola-raimi** [comment]
I just checked it, It has been merged

**Comment by unicodeveloper** [comment]
Alright, I'll close this issue now.
